libs/langchain/langchain/utilities/graphql.py

```diff
@@ -7,14 +7,17 @@
 class GraphQLAPIWrapper(BaseModel):
     """Wrapper around GraphQL API.
 
-    To use, you should have the ``gql`` python package installed.
+    To use, you should have the ``gql`` and ``graphql`` python packages installed.
     This wrapper will use the GraphQL API to conduct queries.
     """
 
     custom_headers: Optional[Dict[str, str]] = None
     graphql_endpoint: str
     gql_client: Any  #: :meta private:
     gql_function: Callable[[str], Any]  #: :meta private:
+    custom_transport_auth: Any  #: :meta private:
+    gql_schema: str
+    auto_fetch_schema: bool
```

Comment:
Ideally this would not be passed into the wrapper, but due to the way the tool build is setup, the value must be passed into here as the wrapper is the only value that receives in the parameters. If instead, it was passed into the graphql tool itself, it would through an error for passing an unused value of "auto_fetch_schema"